internal/analytics/deferred.go

```diff
@@ -122,17 +122,19 @@ func sendDeferred(cfg Configurable, sender func(string, string, string, map[stri
 	if err != nil {
 		return errs.Wrap(err, "Could not load events on send")
 	}
-	for n, event := range deferred {
-		if err := sender(event.Category, event.Action, event.Label, event.Dimensions); err != nil {
-			return errs.Wrap(err, "Could not send deferred event")
+	if len(deferred) > 0 {
```

Comment:
Nit: This is an unnecessary change.

Reply:
It skips the save of the configuration file, which I believe should be avoided if nothing changed.  Of course, it doesn't affect this bug, but as we have seen that viper also has issues with concurrent writes between processes...